Tests/SwiftDocCTests/Model/ParametersAndReturnValidatorTests.swift

```diff
@@ -196,6 +196,39 @@ class ParametersAndReturnValidatorTests: XCTestCase {
         }
     }
     
+    func testFunctionsThatCorrespondToPropertiesInAnotherLanguage() throws {
+        let (_, _, context) = try testBundleAndContext(named: "GeometricalShapes")
+        XCTAssertEqual(context.problems.map(\.diagnostic.summary), [])
+        
+        // A small test helper to format markup for test assertions in this test.
+        func _format(_ markup: [any Markup]) -> String {
+            markup.map { $0.format() }.joined()
+        }
+        
+        let reference = try XCTUnwrap(context.knownPages.first(where: { $0.lastPathComponent == "isEmpty" }))
+        let node = try context.entity(with: reference)
+        
+        let symbolSemantic = try XCTUnwrap(node.semantic as? Symbol)
+        let swiftParameterNames = symbolSemantic.parametersSectionVariants.firstValue?.parameters
+        let objcParameterNames  = symbolSemantic.parametersSectionVariants.allValues.mapFirst(where: { (trait, variant) -> [Parameter]? in
+            guard trait.interfaceLanguage == SourceLanguage.objectiveC.id else { return nil }
+            return variant.parameters
+        })
+        
+        XCTAssertEqual(swiftParameterNames?.map(\.name), [])
+        XCTAssertEqual(objcParameterNames?.map(\.name), ["circle"])
+        XCTAssertEqual(objcParameterNames?.map { _format($0.contents) }, ["The circle to examine."])
+        
+        let swiftReturnsContent = symbolSemantic.returnsSection.map { _format($0.content) }
```

Comment:
minor: This looks to be relying on the fact that Swift will be the default variant, do you think it's worth explicitly searching for the Swift variant in the list of all of them?